src/classBased/components/TodoContainer.js

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import Header from "./Header";
+import TodosList from './TodosList';
+import InputTodo from "./InputTodo";
+import { v4 as uuidv4 } from "uuid";
+
+
+class TodoContainer extends React.Component {
+    state = {
+        todos: []
+       };
+
+    //    componentDidMount() {
+    //        fetch("https://jsonplaceholder.typicode.com/todos?_limit=10")
+    //        .then(response => response.json)
+    //        .then(data => this.setState({todos: data}));
+    //    }
```

Comment:
[OPTIONAL] @jerryowusu  please remove the commented codes to make your code much cleaner

Reply:
Thank you @mwafrika